scripts/api_dump_generator.py

```diff
@@ -369,20 +369,25 @@
 }}
 @end function
 
-VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL api_dump_known_instance_functions(const char* pName)
+VKAPI_ATTR PFN_vkVoidFunction VKAPI_CALL api_dump_known_instance_functions(VkInstance instance, const char* pName)
 {{
     @foreach function where('{funcType}' in ['global', 'instance'] and '{funcName}' not in [ 'vkEnumerateDeviceExtensionProperties' ])
+    @if('${funcDispatchType}' == 'instance')
+    if(strcmp(pName, "{funcName}") == 0 && (!instance || instance_dispatch_table(instance)->{funcShortName}))
+    @end if
```

Comment:
So this causes API Dump to return NULL if the *next* function in the chain is NULL? Seems like a pretty easy way to make sure to only return functions which will successfully pass through the chain (and not crash when called).

Reply:
Just re-read the PR description, it does exactly what I surmised from the change.
And it makes sense, so I'll approve it.